sla_agent.py

```diff
@@ -91,6 +94,50 @@ def __init__(self, skale, node_id=None):
         self.nodes = []
         self.reward_period = call_retry.call(self.skale.constants_holder.get_reward_period)
 
+    def get_last_reward_date(self):
+        node_info = call_retry(self.skale.nodes.get, self.id)
+        return node_info['last_reward_date']
+
+    def generate_monitored_array(self):
+        active_ids = self.skale.nodes.get_active_node_ids()
+
+        active_ids.remove(self.id)
+
+        print(active_ids)
+        if len(active_ids) <= 24:
```

Comment:
what is 24?
add a comment or move to constants

Reply:
these are temporarily added functions, but I moved it to constant anyway